ECOSYSTEM.md

```diff
@@ -4,24 +4,33 @@ We aim to build a vibrant and inclusive ecosystem of partners, core contributors
 
 There are three types of stakeholders:
 
-* **Partners:** Companies that are significantly invested in React Native and have been for years.
+* **Partners:** Companies significantly invested in React Native and take responsibility for the React Native vision and community.
 * **Core Contributors:** Individual people who contribute to the React Native project.
 * **Community Contributors:** Individuals who support projects in the [react-native-community](https://github.com/react-native-community) organization.
 
 ## Partners
 
-Partners are companies that are significantly invested in React Native and have been for years. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Partners think of React Native as a product: they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications.
+Partners are companies that are significantly invested in React Native and demonstrate ownership. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Examples of this may include large scale contributions to `react-native` or owning an essential tools or libraries. 
 
-React Native's current set of partners include Callstack, Expo, Infinite Red, Meta, Microsoft and Software Mansion. Many engineers from these companies are core contributors, and their partner responsibilities also include:
+Partners think of React Native as a product: they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications. 
 
-* **[Callstack](https://callstack.com/):** Manages releases, maintains the [React Native CLI](https://github.com/react-native-community/react-native-cli) and organizes [React Native EU](https://react-native.eu/)
+Baseline responsibilities of partners include: 
+* Attending monthly meeting
+* Contributing to the release process as a [release co-pilot](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2--release-copilot)
```

Comment:
I don't know how we want to deal with cases when partners don't fulfil this requirement but we could decide on this when such a thing happens. I definitely think it's a great first step to make these requirements clear.

Reply:
Yea, I'm not sure either but agree consensus on these responsibilities is a start and if a partner isn't able to meet this, we can have a discussion about why and I think that'll also be helpful

Reply:
the release co-pilot link anchor appears to be broken - i believe it should be updated to:

```suggestion
* Attending monthly meeting
* Contributing to the release process as a [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser)
```

Reply:
expo can certainly help out as a community releaser/release co-pilot in the future, although i wonder if it is useful to require that every partner contribute in this particular way. for example, our role in recent releases has been to provide our feedback from integrating the new release into our tooling and also give technical release support in the form of pull requests and reviews for pull requests. @kudo has been handling this.